README.md

```diff
@@ -1581,35 +1613,6 @@ Selects the rightmost point of each series.
 
 Selects the highest point of each series.
 
-### Plot.select(*selector*, *options*)
-
-Selects the points of each series selected by the *selector*, which can be specified either as a function which receives as input the index of the series, or as a key: value object with exactly one key representing a channel, and the value being a function which receives as inputs the index of the series and the channel or the short-hand "min" and "max" which respectively select the least and greatest points for the specified channel.
-
-For example, to select the point within each series that is the closest to the median of the *y* channel:
-
-```js
-  Plot.select({
-    y: (I, V) => {
-      const median = d3.median(I, i => V[i]);
-      const i = d3.least(I, (i) => Math.abs(V[i] - median));
-      return [i];
-    }, {
-      x: "year",
-      y: "revenue",
-      fill: "format"
-    })
-```
-
-To pick three points at random in each series:
-```js
-  Plot.select(I => d3.shuffle(I).slice(0, 3), {z: "year", ...})
```

Comment:
Note that d3.shuffle(I) mutates in place!

Reply:
Ah, yes. Maybe we should be defensive in `for (const I of Z ? group(facet, i => Z[i]).values() : [facet])` instead, by using `facet.slice()`?

Reply:
or, we could be defensive just in the case that we pass the facet to a custom function (since our internal functions are not mutating I).

Reply:
I don’t think we want to be defensive here because it will introduce overhead. We can document that reducers should not mutate their inputs, if you like (or risk undefined behavior). And we should be careful to not give any examples that mutate their inputs. 🙂